geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/misc/WanLocatorDiscoveryDUnitTest.java

```diff
@@ -65,25 +65,29 @@ private void setupWanSites() throws IOException {
   }
 
   private int setupWanSite1() throws IOException {
-    Properties locator_ln_props = new Properties();
-    // create a cluster
-    locator_ln_props.setProperty(MCAST_PORT, "0");
-    locator_ln_props.setProperty(DISTRIBUTED_SYSTEM_ID, "1");
-
-    locator_ln1 = cluster.startLocatorVM(0, ports[0], locator_ln_props);
-    locator_ln2 = cluster.startLocatorVM(1, ports[1], locator_ln_props, locator_ln1.getPort());
-    return locator_ln1.getPort();
+    locator_ln1 = cluster.startLocatorVM(0, ports[0], VersionManager.CURRENT_VERSION,
+        i -> i.withProperty(MCAST_PORT, "0").withProperty(DISTRIBUTED_SYSTEM_ID, "1"));
```

Comment:
you can get rid of `withProperty(MCAST_PORT, "0")`, it's already set by the rule